packages/devextreme/testing/tests/DevExpress.viz.core/export.tests.js

```diff
@@ -683,10 +737,23 @@ QUnit.test('exportFromMarkup. backgroundColor from current theme', function(asse
     }
 });
 
+QUnit.test('exportWidgets method should pass to export markup as string', function(assert) {
```

Comment:
exportWidgets should pass a dom node to export, not a string, no?

Reply:
yes) you are right. fixed